index.d.ts

```diff
@@ -248,6 +248,15 @@ export function columnResizing(props: {
     cellMinWidth: number,
     view: EditorView,
   ) => NodeView;
+  lastColumnResizable: boolean;
+  updateColumnsOnResize: (
+    node: ProsemirrorNode,
+    colgroup: Element,
+    table: Element,
+    cellMinWidth: number,
+    overrideCol?: number,
+    overrideValue?: number,
+  ) => void;
```

Comment:
How is this different from `updateColumnsOnResize` a few lines down from here?

Reply:
It is not. In fact, it's the same type. I thought of doing `typeof updateColumnsOnResize` but wasn't sure if that would have unintended consequences?